Packs/Core/.secrets-ignore

```diff
@@ -83,4 +83,5 @@ SailPoint
 dummy@dummy.com
 dummy1@dummy.com
 dummy2@dummy.com
-dummy3@dummy.com
\ No newline at end of file
+dummy3@dummy.com
+80.66.75.36
```

Comment:
why you added this IP?

Reply:
cause the PR validation failed on 'secret detection' so I added it to the ignore list.